tests/test_actions.py

```diff
@@ -103,6 +102,31 @@ def test_clean_study(mock_db_config, verbose, prefix, confirm, stats, target, ra
                 assert ("study_valid__456",) not in remaining_tables
 
 
+def test_clean_dedicated_schema(mock_db_config):
+    with mock.patch.object(builtins, "input", lambda _: False):
+        mock_db_config.schema = "dedicated"
+        manifest = study_manifest.StudyManifest("./tests/test_data/study_dedicated_schema/")
+        mock_db_config.db.cursor().execute("CREATE SCHEMA dedicated")
+        builder.run_protected_table_builder(
+            config=mock_db_config,
+            manifest=manifest,
+        )
+        mock_db_config.db.cursor().execute("CREATE TABLE dedicated.table_1 (test int)")
+        mock_db_config.db.cursor().execute(
+            "CREATE VIEW dedicated.view_2 AS SELECT * FROM dedicated.table_1"
+        )
+        cleaner.clean_study(config=mock_db_config, manifest=manifest)
+        remaining_tables = (
+            mock_db_config.db.cursor()
+            .execute("select distinct(table_name) from information_schema.tables")
+            .fetchall()
+        )
+        print(remaining_tables)
```

Comment:
:scissors:

Reply:
all my print ferreting out was on the other PR.

i swear once things have calmed down, https://docs.astral.sh/ruff/rules/print/